### PR TITLE
Gib/qa may 3rd bugfixes

### PR DESCRIPTION
**Pull request by PanopticaRising**

### CODE REVIEW

**Comment by red-tt-bear** [comment]
Package lock has massive amounts of changes while package.json is unchanged

File: src/Components/Quill/QuillControlledEditor.tsx [hunk 1]
```diff
@@ -233,6 +234,9 @@ export const QuillControlledEditor: React.FC<QuillControlledEditorProps> = ({onS
             {onSave && <Grid xs={12} style={{margin: '0% 1%'}}>
                 <Button disabled={disabled} fullWidth color='primary' variant='contained' onClick={onClickedSave}>Save Feedback</Button>
             </Grid>}
+            {!_.isNil(isDirty) && <Grid xs={12} style={{margin: '0% 1%'}}>
```
Comment (red-tt-bear) [inline src/Components/Quill/QuillControlledEditor.tsx:237]: if is dirty isn't passed we hide the button?

**Review by red-tt-bear — CHANGES_REQUESTED** [review]
package lock changes

File: src/Components/Quill/QuillControlledEditor.tsx (hunk 1 above)
Comment (PanopticaRising) [reply src/Components/Quill/QuillControlledEditor.tsx:237]: Yeah, there's `isDirty` which is only used if we're in a RHF and there's `onSave` which is the old behavior that will show a different button (line 235)

**Review by red-tt-bear — APPROVED** [review]

**Pull request merged**